### PR TITLE
Update readme with custom container configs

### PR DESCRIPTION
**Pull request by alapha23**
### Updates 
* Removed Funfile for custom container [issue 3](https://github.com/devsapp/start-puppeteer/issues/3)
* Organized directories for custom container to fix the build failures [issue 4](https://github.com/devsapp/start-puppeteer/issues/4)
* Removed VPC configs because nas auto [issue 5](https://github.com/devsapp/start-puppeteer/issues/5)
* Update readme

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
-```
 
-![alt text](https://github.com/devsapp/start-puppeteer/blob/master/src/vpc.png?raw=true)
+## Deploy using Nodejs 12 with NAS
 
-#### local test
 
 ```
+cd nodejs12
 sudo s build --use-docker
```
Comment (git-qfzhang) [inline readme.md:23]: 这里的 sudo 不是必要的吧？

File: src/nodejs12/.fcignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.s/root
+.s/tmp
```
Comment (git-qfzhang) [inline src/nodejs12/.fcignore:2]: .s/root 和 .s/tmp 应该不会传输的？还有就是你把 node_modules 排除了，是因为这部分要放在 nas 上吗

Comment (alapha23) [reply src/nodejs12/.fcignore:2]: 是的，不放在nas上的话那就大小超了，readme里面我写了 build 之后得 `s nas upload -r -n ./.s/build/artifacts/puppeteer-nodejs12/html2png/.s /mnt/auto`

File: readme.md (hunk 1 above)
Comment (alapha23) [reply readme.md:23]: 好，这儿我都给改了

**Pull request merged**